.github/workflows/deploy-func-convert_ids.yaml

```diff
@@ -0,0 +1,22 @@
+name: Deploy Cloud Functions
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - 'functions/convert_ids/convert_ids_deploy/**'
+
+jobs:
+  deploy-convert-ids-func:
+    uses: ./.github/workflows/helper-deploy-func.yaml
+    with:
+      func-name: "gpz-convert-ids"
+      func-src-dir: "functions/convert_ids/convert_ids_deploy"
+      func-entrypoint: "convert_ids"
+      func-memory-mb: 1024
+      func-data-gcs-url: "gs://geneplexus-func-data/convert-ids/convert-ids_data.tar.gz"
+    permissions:
+      contents: 'read'
```

Comment:
Is this necessary? I think the github_token gets this by default.

Reply:
Good call; I've removed it, and everything works fine, so it wasn't necessary.